SPECS/igc-module.spec

```diff
@@ -0,0 +1,54 @@
+%define uname  %{kernel_version}
+%define module_dir updates
+
+Summary: Driver for igc-module
+Name: igc-module
+Version: 4.20.17
+Release: %{?release}%{!?release:1}
+License: GPL
+Source: %{name}-%{version}.tar.gz
```

Comment:
Please add a comment explaining where the sources come from (the commit message already does, but it will be convenient to have it in the spec file direclty)